classes/components/forms/FieldSlider.php

```diff
@@ -0,0 +1,78 @@
+<?php
+/**
+ * @file classes/components/form/FieldSlider.php
+ *
+ * Copyright (c) 2014-2024 Simon Fraser University
+ * Copyright (c) 2000-2024 John Willinsky
+ * Distributed under the GNU GPL v3. For full terms see the file docs/COPYING.
+ *
+ * @class FieldSlider
+ *
+ * @ingroup classes_controllers_form
+ *
+ * @brief A color picker field in a form.
+ */
+
+namespace PKP\components\forms;
+
+class FieldSlider extends Field
+{
+    /** @copydoc Field::$component */
+    public $component = 'field-slider';
+
+
+    /**
+     * Range min value
+     */
+    public int $min;
```

Comment:
Along with all these control option , I think we should also add the ability to define what is the range steps would be . It should not be a nullable and allow both int and float value .